terraform/aws/README.md

```diff
@@ -0,0 +1,54 @@
+# GCP terraform
+This terraform project creates kubernetes cluster in GCP from scratch, it's required to deploy The Graph indexer on later steps.
+
+## Install prerequisites
+| Tool | Recommended version | Install documentation link |
+| ------------ | ------------ | ------------ |
+| Terraform | `~> 1.0.2` | https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform |
+| Kubectl | Latest | https://kubernetes.io/docs/tasks/tools/#kubectl |
+| AWS cli | Latest | https://cloud.google.com/sdk/docs/install |
+
+## Apply terraform
+### Auth in aws
+Get AWS access key using this guide https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html .
+
+Run in terminal:
+```
+aws configure
+```
+And input Access Key ID and Secret Access Key.
+
+### Fill variables
+Copy example variables file `terraform.example.tfvars` to `terraform.tfvars` in the `terraform/gcp` directory.
+Fill `gcp_project` variable. You also can override other variables defined in `variables.tf`
```

Comment:
```suggestion
Fill `aws_project` variable. You also can override other variables defined in `variables.tf`
```

Reply:
but there is no such variable btw. Also in gcp tf is's called project_id, as I understand